includes/wikia/vignette/VignetteUrlToUrlGenerator.php

```diff
@@ -123,11 +123,11 @@ private function addThumbDefinition(UrlGenerator $generator) {
 		}
 	}
 
-	private function reportError($message, Array $context=[]) {
+	private function reportWarning($message, Array $context=[]) {
 		$exception = new InvalidArgumentException($message);
 		$context['exception'] = $exception;
 
-		$this->error($message, $context);
+		$this->warning($message, $context);
 		throw $exception;
```

Comment:
It will as this method calls (from Vignette class) are wrapped in `try / catch` blocks.